src/language-usability.rst

```diff
@@ -16,6 +16,193 @@ Usability of Type Systems
 API Usability
 =============
 
+..
+  Outline
+  * APIs are ubiquitous
+  * Examples
+  * High level goals of API
+  * Difficulty in achieving goals of API
+  * Distinction with language usability
+..
+
+Application Programming Interfaces (APIs) are interfaces that mediate interactions between software components.
+Examples include software libraries and frameworks, OS system calls, and web protocols.
+Nearly every program written will use an API :cite:`myers2016improving`.
+
+The goal of the API designer is to create an API that a broad range of
```

Comment:
Some APIs are very domain specific while others are intentionally general, so it's not the case that every API aims to serve a broad range of programmers. See if you can make that distinction more clear here.